packages/client/src/runtime/core/types/GetResult.ts

```diff
@@ -50,7 +52,7 @@ export type GetFindResult<P extends Payload, A> =
             : K extends '_count'
               ? Count<GetFindResult<P, S[K]>>
               : never
-  } & (A extends { include: any } ? P['scalars'] : unknown)
+  }& (A extends { include: any } & Record<string, unknown> ? P['scalars'] : unknown)
```

Comment:
Fixes an issue with result type inference when `include` + another key is used (eg. `include` + `where` (see added test))